tests/screen-objects/hiltonMobileAppPage.js

```diff
@@ -0,0 +1,31 @@
+import BasePage from './basePage'
+
+class HiltonMobileAppPage extends BasePage {
+
+	constructor() {
+		super();
+	}
+
+	open() {
+		//do the operations that will open the hilton mobile app
+	}
+
+	isLoaded() {
+		/**
+		 * This is the new preferred way to interact with elements
+		 * and makes the migration to wdio V5 easier because the above method
+		 * will be removed from V5 (no selectors are allowed anymore with
+		 * actions)
+		 */
+		const header = $('#global_header');
```

Comment:
I would not place selectors in the methods, I'd advice to have them at the top of a PO because they will give you overview and are likely to be changed the most